cylc/uiserver/app.py

```diff
@@ -463,13 +465,17 @@ def initialize_settings(self):
         super().initialize_settings()
 
         # startup messages
-        self.log.info("Starting Cylc UI Server")
-        self.log.info(f'Serving UI from: {self.ui_path}')
+        self.log.info(f"Starting Cylc UI Server v{__version__}")
+        self.log.info(f"Serving UI from: {self.ui_path}")
+
+        try:
+            self.log.setLevel(self.log_level)
+        except Exception as exc:
+            self.log.warning(exc)
```

Comment:
I'm not sure whether this is the correct behaviour for the `log_level` trait. It's a bit complicated, but it is used [by traitlets] as the default log level for terminal output, not for the logger itself.